src/pe_reports/stakeholder/views.py

```diff
@@ -0,0 +1,607 @@
+
+# Third-Party Libraries
+from flask import Blueprint, flash, redirect, render_template, url_for
+
+# from flask import Flask, flash, redirect,request, render_template, url_for
+import psycopg2
+import psycopg2.extras
+import requests
+
+# cisagov Libraries
+# Local file import
+from pe_reports.data.config import config
+from pe_reports.stakeholder.forms import InfoFormExternal
+
+logging.basicConfig(
+    filemode="a",
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S",
+    level=logging.INFO,
+)
+
+# CSG credentials
+API_Client_ID = os.getenv("CSGUSER")
+API_Client_secret = os.environ.get("CSGSECRET")
+API_WHOIS = os.environ.get("WHOIS_VAR")
+
+conn = None
+cursor = None
+thedateToday = date.today().strftime("%Y-%m-%d")
+
+
+def getToken():
+    """Will get authorization token from CSG."""
+    d = {
+        "grant_type": "client_credentials",
+        "client_id": f"{API_Client_ID}",
+        "client_secret": f"{API_Client_secret}",
+    }
+    r = requests.post("https://api.cybersixgill.com/auth/token", data=d)
+    r = r.text.split(":")
+    r = r[1].lstrip('"').rsplit('"')[0]
+    return r
+
+
+def getAgencies(org_name):
+    """Get all agency names from P&E database."""
+    global conn, cursor
+    resultDict = {}
+    try:
+        params = config()
+
+        conn = psycopg2.connect(**params)
+
+        if conn:
+            logging.info(
+                "There was a connection made to"
+                "the database and the query was executed. "
+            )
+
+            cursor = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+
+            query = "select organizations_uid,name from"
+            f" organizations where name='{org_name}';"
+
+            cursor.execute(query)
+
+            result = cursor.fetchall()
+
+            for row in result:
+                theOrgUUID = row[0]
+                theOrgName = row[1]
+
+                resultDict[f"{theOrgUUID}"] = f"{theOrgName}"
+            return resultDict
+
+    except (Exception, psycopg2.DatabaseError) as err:
+        logging.error(f"There was a problem logging into the psycopg database {err}")
+    finally:
+        if conn is not None:
+            cursor.close()
+            conn.close()
+            logging.info("The connection/query was completed and closed.")
+
+            return resultDict
+
+
+def getRootID(org_UUID):
+    """Get all agency names from P&E database."""
+    global conn, cursor
+    resultDict = {}
+    try:
+        params = config()
+
+        conn = psycopg2.connect(**params)
+
+        if conn:
+            logging.info(
+                "There was a connection made to the database and the query was executed "
+            )
+
+            cursor = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+            query = "select root_domain_uid, organization_name from"
+            f" root_domains where organizations_uid='{org_UUID}';"
+
+            cursor.execute(query)
```

Comment:
```suggestion
            query = "select root_domain_uid, organization_name from"
            " root_domains where organizations_uid='{}';"

            cursor.execute(query.format(org_UUID))
```

Reply:
Want any variable replacements to occur inside of the execute function to avoid injection.